Makefile.PL

```diff
@@ -4,7 +4,7 @@ pdlmaker_init();
 
 ##-- preqrequistes (for PREREQ_PM)
 my %prereq = (
-	      'PDL' => 0, #'2.4.2',
+	      'PDL' => '2.066',
```

Comment:
I'd like to remain nominally compatible at least with v2.019 (stock debian buster version) for the moment.